frontend/tests/controllers-test/dashCtrl.test.js

```diff
@@ -0,0 +1,282 @@
+'use strict';
+
+describe('Unit tests for Dashboard Controller', function () {
```

Comment:
`Unit tests for Dashboard Controller` -> `Unit tests for dashboard controller`

Reply:
done